lib/vagrant-openshift/action/install_openshift_base_dependencies.rb

```diff
@@ -104,11 +104,18 @@ def call(env)
 
 if [[ ${GO_VERSION[2]} == "go1.4"* ]]; then
   GOPATH=/data go get golang.org/x/tools/cmd/cover
-else
-  GOPATH=/data go get code.google.com/p/go.tools/cmd/cover
-fi
 
-GOPATH=/data go get golang.org/x/tools/cmd/vet
+  GOPATH=/data go get golang.org/x/tools/cmd/vet
+
+  # TODO: Remove for go1.5
```

Comment:
Move the todo up to the line with the "if 1.4*" check… that's what would need to be removed